action.yaml

```diff
@@ -49,35 +54,52 @@ runs:
         fetch-depth: ${{ inputs.REPO_DEPTH }}
         ref: ${{ inputs.VERSION_TAG }}
         repository: ${{ inputs.REPOSITORY }}
+    
     - name: Git Config
       run: |
         git config --global user.name "GitHub Actions"
         git config --global user.email "actions@users.noreply.github.com"
       shell: bash
+
+    - name: Install pnpm
+      if: inputs.NODE_PACKAGE_MANAGER == 'pnpm'
+      uses: pnpm/action-setup@v3
+      with:
+        version: 8
```

Comment:
Does this need to be configurable? I know different repos are using different versions, are you assuming we're going to make them all use the same version now?

Reply:
I can remove this to relying on the `packageManager` field in the package.json, but you must use one or the other, https://github.com/pnpm/action-setup/blob/master/src/install-pnpm/run.ts#L36-L59. If someone is using a version less than 8 that is a problem as version 8 has existed for a year.

Reply:
Okay, I thought one our repos had issues but I just looked and it seems they're all on 8 now. Maybe it was an issue with the default version from an older node being different. But yeah if it can use the one from the project that's even better